conf/application.conf

```diff
@@ -60,6 +62,16 @@ auditing {
   enabled = true
 }
 
+mongodb {
+  uri = "mongodb://localhost:27017/"${appName}
+  encryption.key = "QmFyMTIzNDVCYXIxMjM0NQ=="
+  timeToLive = 15minutes
+  journeyAnswersTimeToLive = 30days
+  replaceJourneyAnswersIndexes = false
+}
+
+useEncryption = false
```

Comment:
Similarly, this is used in State Benefits because of pre-existing encryption logic within the service and isn't required here